src/infra/src/table/dashboards.rs

```diff
@@ -191,6 +192,7 @@ pub async fn put(
         .map(|d| d.to_owned());
     let version = dashboard.version;
     let created_at_depricated = dashboard.created_at_deprecated();
+    let updated_at = dashboard.updated_at;
```

Comment:
I think we probably should be generating the updated_at timestamp here rather than expecting the client to set it.

Reply:
@patrickcsullivan yes ideally, the updated_at should have been updated here. But since the super cluster directly calls this function and for super cluster the updated_at must be same as the passed `updated_at`value (not the current timestamp). But, I think I can change the function signature a little bit to make both of use happy :)